freqtrade/tests/test_backtesting.py

```diff
@@ -1,36 +1,84 @@
 # pragma pylint: disable=missing-docstring
+import json
 import logging
 import os
+from typing import Tuple, Dict
 
-import pytest
 import arrow
+import pytest
+from arrow import Arrow
 from pandas import DataFrame
+from tabulate import tabulate
 
 from freqtrade import exchange
 from freqtrade.analyze import analyze_ticker
 from freqtrade.exchange import Bittrex
 from freqtrade.main import should_sell
 from freqtrade.persistence import Trade
+from freqtrade.tests import load_backtesting_data
+
+logger = logging.getLogger(__name__)
 
-logging.disable(logging.DEBUG)  # disable debug logs that slow backtesting a lot
 
+def format_results(results: DataFrame):
+    return 'Made {} buys. Average profit {:.2f}%. ' \
+           'Total profit was {:.3f}. Average duration {:.1f} mins.'.format(
+               len(results.index),
+               results.profit.mean() * 100.0,
+               results.profit.sum(),
+               results.duration.mean() * 5,
+           )
```

Comment:
I would actually prefer to have simple unit tests for these too. It seems trivial but at the same time it's important that we don't make mistakes and show let's say the wrong total profit.